paasta_tools/deployd/master.py

```diff
@@ -126,7 +126,10 @@ def startup(self) -> None:
         self.log.info(
             "All watchers started, now adding all services for initial bounce"
         )
-        self.add_all_services()
+
+        # Fill the queue if we are not using the persistent ZK queue
+        if not self.config.get_deployd_use_zk_queue():
```

Comment:
What do you think about removing the feature toggle complexity soon?

Reply:
I guess I was figuring that deployd will go away when Mesos goes away and we'd leave it till then?  I don't have a strong opinion though.